src/playlist/PlaylistSong.cxx

```diff
@@ -54,18 +56,22 @@ merge_song_metadata(DetachedSong &add, const DetachedSong &base) noexcept
 }
 
 static bool
-playlist_check_load_song(DetachedSong &song, const SongLoader &loader) noexcept
-try {
-	DetachedSong tmp = loader.LoadSong(song.GetURI());
-
-	song.SetURI(tmp.GetURI());
-	if (!song.HasRealURI() && tmp.HasRealURI())
-		song.SetRealURI(tmp.GetRealURI());
-
-	merge_song_metadata(song, tmp);
-	return true;
-} catch (...) {
-	return false;
+playlist_check_load_song(DetachedSong &song, const SongLoader &loader) noexcept {
+	try {
+		DetachedSong tmp = loader.LoadSong(song.GetURI());
+
+		song.SetURI(tmp.GetURI());
+		if (!song.HasRealURI() && tmp.HasRealURI())
+			song.SetRealURI(tmp.GetRealURI());
+
+		merge_song_metadata(song, tmp);
+		return true;
+	} catch (const std::exception &ex) {
+		FmtWarning(playlist_domain,
```

Comment:
If your playlist has a million songs that don't exist, this will produce a million log lines. That's why I never implemented this one - I feared this would spam the logs due to loading a broken playlist from the internet.

Reply:
OK, I see your point. Still, failing silently seems not ideal, either. Would you be open to logging at a lower verbosity? Or, what I think I would prefer, moving the log message further up and including logic to prevent this: log the first, say, 8 missing files, then a message along the lines of "further errors elided..."